packages/common/src/eips.ts

```diff
@@ -505,6 +505,13 @@ export const EIPs: EIPsDict = {
       },
     },
   },
+  6110: {
+    comment: 'Supply validator deposits on chain',
+    url: 'https://eips.ethereum.org/EIPS/eip-6110',
+    status: Status.Draft,
+    minimumHardfork: Hardfork.Cancun,
```

Comment:
The `minimumHardfork` is not the inclusion HF, but the first HF from which this EIP can be run in isolation (so: all pre-requisits for the EIP are met). So we can (should) switch to `Cancun` here, just as some additional context.